instantsearch.js/refresh-cache-periodically/src/app.js

```diff
@@ -22,16 +22,16 @@ search.addWidgets([
   instantsearch.widgets.hits({
     container: '#hits',
     templates: {
-      item: `
+      item: (hit, { html, components }) => html`
         <div>
           <header class="hit-name">
-            {{#helpers.highlight}}{ "attribute": "name" }{{/helpers.highlight}}
+            ${components.Highlight({ hit, attribute: 'name' })}
           </header>
-          <img src="{{image}}" align="left" />
+          <img src="${hit.image}" align="left" />
           <p class="hit-description">
-            {{#helpers.highlight}}{ "attribute": "description" }{{/helpers.highlight}}
+            ${components.Highlight({ hit, attribute: 'description' })}
           </p>
-          <p class="hit-price">\${{price}}</p>
+          <p class="hit-price">\$${hit.price}</p>
```

Comment:
I think `$` sign was escaped and we do not need the `\` anymore ?

Reply:
This was for appending a '$' before the price which is just a number/float value

Reply:
doesn't it still need to be escaped for a literal $?

Reply:
Yeah but is the backslash before still needed with `html` ?